src/views/Deposit/index.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import 'styled-components/macro';
```

Comment:
Don't we need to do `import type {} from 'styled-components/cssprop';` here too?

Reply:
Oh, I did a variation of that here, and looks like it did the trick: https://github.com/pokt-network/wpokt-app/pull/5/commits/9bff08bcf588badca595e56b8f06e8cfde1502d0#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R4

Is it giving you errors still?

Reply:
Haven't tried locally haha! Just noticed the discrepancy between both files and wanted to make sure the codebase's in sync.

Reply:
Just double-checked and works fine 👌

Reply:
Then let's standardize on either importing the macro only or the `cssprop` as well 😄

Reply:
Can do!

Just want to point out, though, that the docs are saying it's only necessary to put it in once: https://styled-components.com/docs/api#usage-with-typescript.

Do you still want me to import `cssprop` with `macro` each time for the sake of standardization?